cookbook/product/query-products.rst

```diff
@@ -81,7 +82,7 @@ The PQB uses the registry to resolve the filter to use.
 
 A filter can be used on field (means on doctrine fields of product mapping as id, family, etc) or on attribute (means on product value, as a sku, a name, etc).
 
-To add your own filter, you need to implement a class implementing FieldFilterInterface and/or AttributeFilterInterface and declare a service as :
+To add your own filter, you need to implement a class implementing ``FieldFilterInterface`` and/or ``AttributeFilterInterface`` and declare a service as :
```

Comment:
"implement a class implementing" is weird. 
"you need to create a class implementing" ?